app/refresh-ingestion.sh

```diff
@@ -155,13 +155,32 @@ echo_stats(){
     echo "Markdown file count: $(find "${DATASET_ID}-${TODAY}_md" -type f -iname '*.md' | wc -l)"
     echo "-----------------------------------"
     echo "REMINDERS:"
-    echo "1. Upload the zip file to the 'Chatbot Knowledge Markdown' Google Drive folder, replacing the old zip file."
+    echo_cmds
+}
+
+echo_cmds(){
+    local DATASET_ID="$1"
+    echo "# 1. Upload the zip file to the 'Chatbot Knowledge Markdown' Google Drive folder, replacing the old zip file."
     echo "   $(ls -l "${DATASET_ID}_md.zip")"
-    echo "2. Upload ingester input files (e.g., *-scrapings.json) and stats to S3:"
-    echo "   aws s3 sync src/ingestion/${DATASET_ID}_scrapings*.json s3://decision-support-tool-app-dev/${DATASET_ID}"
-    echo "   aws s3 cp logs/${DATASET_ID}-${TODAY}_stats.json s3://decision-support-tool-app-dev/${DATASET_ID}/stats/${TODAY}_stats.json"
-    echo "3. Run ingestion on deployed app:"
-    echo "   ./bin/run-command app dev ..."
+    echo ""
+    echo "# 2. Upload ingester input files (e.g., *-scrapings.json) and stats to S3:"
+
+    if [ "$DATASET_ID" == "imagine_la" ]; then
+        local S3_HTML_DIR="s3://decision-support-tool-app-dev/imagine_la-${TODAY}"
```

Comment:
Do you want to also dynamically set the S3 bucket by `$DEPLOY_ENV` here too? (if so lines 174 and 176 will also need to be updated)

Reply:
Do you mean:
```suggestion
        local S3_HTML_DIR="s3://decision-support-tool-app-${DEPLOY_ENV}/imagine_la-${TODAY}"
```
If so, I was wondering of that. I'm guessing `prod` cannot access `s3://decision-support-tool-app-dev`. I'll make the change.